src/peft/peft_model.py

```diff
@@ -252,7 +254,7 @@ def save_pretrained(
                         output_state_dict[shared_tensor_name] = output_state_dict[shared_tensor_name].clone()
 
                 safe_save_file(
-                    output_state_dict,
+                    {k:v.contiguous() for k,v in output_state_dict.items()},
```

Comment:
Why do we need the `contiguous` calls here? There should be a comment to explain this. Do you know if this could affect other models negatively?

Reply:
To avoid side effects, I chaged codes to call `contiguous` in `get_peft_model_state_dict`.